### DISCUSSION TITLE
Screenshot of UI in repo does not match the UI in deployed docker image

### DISCUSSION THREAD

**Discussion by EMjetrot** [mckaywrigley/chatbot-ui#1585] [Help]
Hi - 

First and foremost a great thanks to creator and the contributors to this great repo :)

I recently deployed the docker image of Chatbot-UI in Azure using the package https://github.com/mckaywrigley/chatbot-ui/pkgs/container/chatbot-ui, but the UI in the deployed container doesn't seem to match the one shown in the repo. Is the packaged image an older version and if so, are any updates planned?

Here is a screenshot of the UI i'm seeing when deployed on Azure:

![screenshot](https://github.com/mckaywrigley/chatbot-ui/assets/130550972/48c7bcc0-0da2-4cee-975f-42668d1cf398)

Here is the screenshot in the repo:

<img width="1153" alt="screenshot_repo" src="https://github.com/mckaywrigley/chatbot-ui/assets/130550972/714dcdf0-bba4-4dbc-a056-ad91bf446a2d">

Kind regards

Jens

**Comment by secondtruth** [comment]
Yes, the container image is for the legacy version.
The version from the screenshot in the readme is also available, but the currently only officially supported installation method is the one described in the readme. But there is a community approach to support a fully dockerized installation: #1193

**Reply by EMjetrot** [reply]
Okay - thank you very much for the explanation and the link :)
